i18n/translations/de-AT.json

```
@@ -0,0 +1,138 @@
+{
+    "copy-to-clipboard": "In die Zwischenablage kopieren",
+    "author-anonymous": "Anonymus",
+    "action-bar-action": {
+        "visit-home-page": "Auf die Homepage gehen",
+        "open-an-issue": "Einen Fehler melden",
+        "download-tarball": "Tarball Herunterladen"
```

```suggestion
        "download-tarball": "Tarball herunterladen"
```

I'm not quite sure if it's usual to say "tarball" in German, you could also just say `Archiv herunterladen`.